src/NumericalAlgorithms/Interpolation/InterpolationTargetReceiveVars.hpp

```diff
@@ -136,16 +136,22 @@ void fill_invalid_points(const gsl::not_null<db::DataBox<DbTags>*> box,
   }
 }
 
-/// Calls the callback function, tells interpolators to clean up the current
-/// temporal_id, and then if there are more temporal_ids to be interpolated,
-/// starts the next one.
+/// Prepares for calling the callback function on an InterpolationTarget,
+/// calls the function, and returns true if the callback is done and
+/// false if the callback is not done (e.g. if the callback is an
+/// apparent horizon search and it needs another iteration).
```

Comment:
To me this would be easier to parse as (assuming I got it right, which I might not have!):
```
Wraps calling the callback function on an InterpolationTarget.

First prepares for the callback, then calls the callback,
and returns true if ... 
```

Reply:
:+1: